libsolidity/analysis/StaticAnalyzer.cpp

```diff
@@ -20,14 +20,17 @@
  * Static analyzer and checker.
  */
 
-#include <libsolidity/analysis/StaticAnalyzer.h>
 #include <memory>
```

Comment:
We are in process of changing the header order to:

header that belongs to the cpp file

headers of the sub-project
headers of the project
boost
std

(i.e. in direction from most specific to most generic with an exception for the header belonging to the cpp file)

This has the advantage that some dependency problems between headers are more likely to be detected.

You don't need to change this here, though, if you don't want to.